conda.recipe/test_bdist_conda_setup.py

```diff
@@ -0,0 +1,8 @@
+from distutils.core import setup, Extension
+import distutils.command.bdist_conda
```

Comment:
Is this needed?

Reply:
good point... that is the old test file from the conda-forge-feedstock... those import are not required / can be changed for setup.py entry_point.

Reply:
Thanks for pointing me at that... with the entry_point it is now enough to do a `from setuptools import setup` ... that gets it working.